package.json

```
@@ -275,6 +276,18 @@
                   "type": "string",
                   "description": "%cosmywasmy.chains.items.defaultGasPrice.desc%"
                 },
+                "chainGasDenom": {
+                  "type": "string",
+                  "description": "%cosmywasmy.chains.items.chainGasDenom.desc%"
+                },
+                "chainDenomDecimals": {
+                  "type": "string",
+                  "description": "%cosmywasmy.chains.items.chainDenomDecimals.desc%"
+                },
+                "signType": {
+                  "type": "string",
```

```suggestion
                  "type": "string",
                  "enum": ["ethsecp256", "tmsecp256"],
                  "default": "tmsecp256",
```
Can you turn this config into enum?